### PR TITLE
Backport PR 2789 to v3.9.x (Orientation with empty viewer)

### PR DESCRIPTION
**Pull request by rosteen**
Forgot to add backport label to original PR. #2789

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -15,7 +15,7 @@ Imviz
 ^^^^^
 - Fix a bug where footprints did not overlay when created via API. [#2790]
 
-- Improved behavior when orientations are created or selected without having data loaded in the viewer. [#2789]
+- Improved behavior when orientations are created or selected without having data loaded in the viewer. [#2808]
```
Comment (kecnry) [inline CHANGES.rst:18]: I think the bot just leaves this referencing the original PR and then just add the no changelog entry needed label

Comment (rosteen) [reply CHANGES.rst]: Makes sense, fixed!

**Review by kecnry — APPROVED** [review]

**Comment by pllim** [comment]
For future reference, unless there is conflict, you could have invoked the bot via comment too. Please see https://meeseeksbox.github.io/ (scroll down a bit). FYI

**Comment by rosteen** [comment]
Test failure is unrelated, merging.

**Pull request merged**